tiflash: remove tiflash tar (#2315)
pingcap/docs: reference/tiflash/deploy.md

For fresh TiFlash deployment, it is recommended to deploy TiFlash by downloading an offline installation package. The steps are as follows:

> 1. Download TiDB Ansible corresponding to the tag of the TiDB 3.1 version:

Reviewer: ```suggestion
1. Download TiDB Ansible with the specified tag corresponding to the TiDB 3.1 version:
```

    {{< copyable "shell-regular" >}}

    ```shell
    git clone -b $tag https://github.com/pingcap/tidb-ansible.git
    ```

>  2. Download the binary files of TiDB, TiKV, PD components:

Reviewer: ```suggestion
 2. Download the binary files of TiDB, TiKV, PD, and other components:
```

>  2. Execute the following `local prepare` command to download the binary files of TiDB, TiKV, PD components:

Reviewer: ```suggestion
 2. Download the binary files of TiDB, TiKV, PD components:
```
